shared/chat/conversation/input-area/suggestors/suggestion-list.desktop.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import * as React from 'react'
+import ReactList from 'react-list'
+import * as Kb from '../../../../common-adapters'
+import * as Styles from '../../../../styles'
+import type {Props} from './suggestion-list'
+
+class SuggestionList extends React.Component<Props> {
+  _listRef = React.createRef<ReactList>()
+
+  componentDidMount() {
+    // hack to get `ReactList` to render more than one item on initial mount
```

Comment:
is this a bug in ReactList?

Reply:
we should really start to move away from react-list and use react-window instead

Reply:
it is a bug there. may be also what's plaguing the stellar search suggestions cc @zanderz ?